community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -625,10 +626,15 @@ public void visitDeletedNode( long id )
         {
             try
             {
+                counts.incrementNodeCount( ANY_LABEL, -1 );
                 PrimitiveIntIterator labels = storeLayer.nodeGetLabels( id );
```

Comment:
No, that path is not used during recovery, this is the central part of transforming transaction state to commands that will then go into the log, so it is very much part of "live operations" only.

Sent from a device with virtual keyboard, please excuse typos and brevity.

> On 26 feb 2015, at 20:25, Chris Vest <notifications@github.com> wrote:
> 
> In community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java:
> 
> > @@ -625,10 +626,15 @@ public void visitDeletedNode( long id )
> >          {
> >              try
> >              {
> > +                counts.incrementNodeCount( ANY_LABEL, -1 );
> >                  PrimitiveIntIterator labels = storeLayer.nodeGetLabels( id );
> This code path isn't used during recovery, right?
> 
> —
> Reply to this email directly or view it on GitHub.
>